scm/doc/TechGuide/chap_hsd.tex

```diff
@@ -0,0 +1,212 @@
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/etc/scripts/ccpp_scheme_sim
+./create_1D_CSSdata.py --cases twpice --suites SCM_GFS_v17_p8 --time 3600
+\end{lstlisting}
+\end{enumerate}
+
+The data file will be written to \execout{ccpp-scm/scm/etc/scripts/ccpp\_scheme\_sim/} with the following format, \execout{data\_scheme\_sim\_DIM.CASES.SUITES.nc}.
+
+\subsection{Example 1: Suite with Active Radiation}
+\label{section:Suite_with_Active_Radiation}
+
+For this example we will use the two-dimensional forcing data from \ref{section:Creating_Custom_Data_for_Simulator}. 
+
+First, we need to modify the SDF to include the CSS, \execout{ccpp\_suite\_simulator.F90} and an additional interstital scheme to couple to the GFS physics, \execout{GFS\_ccpp\_suite\_sim\_pre.F90} (See \ref{fig:CSS_SDF_ex1}).
+
+\begin{figure}[t]
+    \centering
+    \includegraphics[width=0.80\textwidth]{images/SDF_changes_for_CSS_ex1.png}
+    \caption{Changes to GFS v16 physics SDF to include CCPP suite simulator for \execout{active} radiation parameterization. All other parameterizations are replaced by simulated data.}
+    \label{fig:CSS_SDF_ex1}
+\end{figure}
+
+Next, the physics namelist needs to be configured to:
+\begin{enumerate}
+\item Add data file, \execout{suite\_sim\_file} created in \ref{section:Creating_Custom_Data_for_Simulator} to the namelist.
+\item Turn ``off'' all schemes except the radiation (see Fig. \ref{lst_css_nml})
+\end{enumerate}
+
+Finally, we rebuild the SCM with the modified SDFs to include the CSS, and run the SCM  using TWPICE case with the modified \execout{GFS\_v16} suite.
+\begin{lstlisting}[language=bash]
+cd ccpp-scm/scm/bin
+cmake ../src -DCCPP_SUITES=SCM_GFS_v16
+./run_scm.py -c twpice -s SCM_GFS_v16
+\end{lstlisting}
+
+\subsection{Example 2: Suite with Active Cloud Microphysics}
+\label{section:Suite_with_Active_cldmp}
+
+For this example we will use the constant forcing data from \ref{section:Creating_Custom_Data_for_Simulator}.
+
+First, we need to modify the SDF to include the CSS, \execout{ccpp\_suite\_simulator.F90} and an additional interstital scheme to couple to the GFS physics, \execout{GFS\_ccpp\_suite\_sim\_pre.F90} (See \ref{fig:CSS_SDF_ex2}).
+
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.80\textwidth]{images/SDF_changes_for_CSS_ex2.png}
+    \caption{Changes to GFS v17 prototype 8  physics SDF to include CCPP suite simulator for \execout{active} cloud microphysics parameterization. All other parameterizations are replaced by simulated data.}
+    \label{fig:CSS_SDF_ex2}
+\end{figure}
+
+Next, the physics namelist needs to be configured to:
+\begin{enumerate}
+\item Add data file, \execout{suite\_sim\_file} created in \ref{section:Creating_Custom_Data_for_Simulator} to the namelist.
+\item Turn ``off'' all schemes except the cloud microphysics (see Fig. \ref{lst_css_nml_ex2})
+\end{enumerate}
+
+\lstinputlisting[
```

Comment:
suite_sime_file uses old naming convention > should be '...ccpp_suite_sim/data_CSS_1D.twpice.SCM_GFS_v16.nc'
Again, since you created the sim data with p8, the suite name is different. Might use all the same suite for consistency. Or maybe 1D is v16, and 2D is p8...